packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -584,27 +604,29 @@ export class AssetGraphBuilder {
 
   propagateSymbolsUp(
     visit: (
-      node: AssetNode,
+      nodeId: NodeId,
```

Comment:
Can you pass in the `AssetNode` as before here and retrieve the node in `propagateSymbolsUp` (and not in the callback)?